External/Plugins/FlashDebugger/Helpers/CopyAsTree.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using Aga.Controls.Tree;
+using FlashDebugger.Controls;
+
+namespace FlashDebugger
+{
+	public class CopyTreeHelper
+	{
+
+		public const int CopyTreeMaxRecursion = 10;
+		public const int CopyTreeMaxChars = 1000000;
+
+		private static Dictionary<string, int> CopiedObjects;
```

Comment:
CopyTreeHelper.CopiedObject is static, and right now after the function exits it isn't freed, I think it would be better to just pass it as an argument to GetTreeItemsAsText, it's only used there.